File: matiasmasso/Mat.NET/DT2/DTOInsolvencia.vb
Public Class DTOInsolvencia
    Inherits DTOBaseGuid

    Property Customer As DTOCustomer
    Property Fch As Date
    Property Nominal As DTOAmt

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Shared Function Factory() As DTOInsolvencia
        Dim retval As New DTOInsolvencia
        With retval
            .Fch = DateTime.Today
        End With
        Return retval
    End Function

End Class

File: matiasmasso/Mat.NET/DTO/DTOInsolvencia.cs
using System;

namespace DTO
{
    public class DTOInsolvencia : DTOBaseGuid
    {
        public DTOCustomer Customer { get; set; }
        public DateTime Fch { get; set; }
        public DTOAmt Nominal { get; set; }

        public DTOInsolvencia() : base()
        {
        }

        public DTOInsolvencia(Guid oGuid) : base(oGuid)
        {
        }

        public static DTOInsolvencia Factory()
        {
            DTOInsolvencia retval = new DTOInsolvencia();
            {
                var withBlock = retval;
                withBlock.Fch = DTO.GlobalVariables.Today();
            }
            return retval;
        }
    }
}
